Framework/DataHandling/src/LoadMuonStrategy.cpp

```diff
@@ -124,5 +124,28 @@ DataObjects::TableWorkspace_sptr LoadMuonStrategy::createDeadTimeTable(
   return deadTimesTable;
 }
 
+/**
+ * Creates a timezero table for the loaded detectors
+ * @param numSpec :: Numer of spectra (number of rows in table)
+ * @param timeZeros :: Vector containing time zero values for each spectra
+ * @return TableWorkspace of time zeros
+ */
+DataObjects::TableWorkspace_sptr LoadMuonStrategy::createTimeZeroTable(
+    const size_t numSpec, const std::vector<double> &timeZeros) const {
+  Mantid::DataObjects::TableWorkspace_sptr timeZeroTable =
+      std::dynamic_pointer_cast<Mantid::DataObjects::TableWorkspace>(
+          Mantid::API::WorkspaceFactory::Instance().createTable(
+              "TableWorkspace"));
+  assert(timeZeroTable);
```

Comment:
it should exist as we just made it. This type of assert can be dangerous as it will propagate an error upwards